src/pages/ExportCSV/ExportCSV.jsx

```diff
@@ -0,0 +1,303 @@
+import React from 'react';
+import { ChakraProvider, Select, Button, Input, FormLabel } from '@chakra-ui/react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+import './ExportCSV.css';
+import { isValidZip } from '../../common/utils';
+
+function ExportCSV() {
+  const [sortOption, setSortOption] = React.useState('ascend');
+  const [boxesOption, setBoxesOption] = React.useState('all');
+  const [dateOption, setDate] = React.useState('all');
+  const [launchOption, setLaunchOption] = React.useState('both');
+  const [zipOption, setZipOption] = React.useState('all');
+  const [customBoxes, setCustomBoxes] = React.useState('');
+  const [customDate, setCustomDate] = React.useState(new Date());
+  const [customStartDate, setCustomStartDate] = React.useState('');
+  const [customEndDate, setCustomEndDate] = React.useState('');
+  const [customZip, setCustomZip] = React.useState('');
+
+  // Function to check valid range
+  function checkRange(value) {
```

Comment:
Change this and the other functions in the file to be arrow functions, so we can keep things consistent and cleaner

Reply:
finished